### PR TITLE
event comment modal bug patch

### PR DESCRIPTION
**Pull request by royce-reed**

### CODE REVIEW

**Review by iAmGjert — APPROVED** [review]

File: database/index.ts [hunk 1]
```diff
@@ -447,159 +447,84 @@ db.sync(
...
-                      user_id: 3,
-                    },
-                    {
-                      id: 4,
-                      event_id: 4,
-                      user_id: 4,
-                    },
-                  ]);
-                })
-                .then(()=>{
```
[the remark this answers is not in the document]
Comment (iAmGjert) [reply database/index.ts:519]: does this push not have the job data? I guess I don't need it anymore, but theres still a few bugs on the page that need to be fixed. I think I had it working to not display jobs you created yourself, and not display jobs that already have an assigned sitter
I was having issues getting it to not display jobs you've already applied for

[the remark this answers is not in the document]
Comment (bford002) [reply database/index.ts:519]: I think I would still need them. I guess depending on how well the job creating is working right now.

[the remark this answers is not in the document]
Comment (Ibenyourbro) [reply database/index.ts:519]: Im also looking at lines 520 -587. Do the jobs need that data?

[the remark this answers is not in the document]
Comment (royce-reed) [reply database/index.ts:519]: this has been resolved

**Review by bford002 — APPROVED** [review]

**Pull request closed without merging**